nanoFramework.Graphics/nanoFramework.Graphics.nfproj

```diff
@@ -277,9 +276,8 @@
     <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="mscorlib, Version=1.14.3.0, Culture=neutral, PublicKeyToken=c07d481e9758c731">
-      <HintPath>..\packages\nanoFramework.CoreLibrary.1.14.2\lib\mscorlib.dll</HintPath>
-      <Private>True</Private>
+    <Reference Include="mscorlib">
```

Comment:
I'm not very keen on having all these refs without the public key... 
Sometime ago I recall having to edit a bunch of project files manually to fix that.
Can you please give it a try and re-add these to fix this?
Are you using VS or what?